dev/reference/garbage-collection.md

```diff
@@ -2,7 +2,8 @@
 title: TiDB Garbage Collection (GC)
 summary: Use Garbage Collection (GC) to clear the obsolete data of TiDB.
 category: reference
-aliases: ['/docs/op-guide/gc/'] 
+aliases: ['/docs/op-guide/gc/']
+exists_in_docs_cn: false
```

Comment:
This file does exist in docs-cn, just the existing file name is not consistent with that in docs. I've updated it in https://github.com/pingcap/docs-cn/pull/1401.